hubstorage/client.py

```diff
@@ -162,6 +170,16 @@ def close(self, timeout=None):
         if self._batchuploader is not None:
             self.batchuploader.close(timeout)
 
+    @property
+    def USERAGENT(self):
+        warnings.warn(USERAGENT_DEPRECATED_MSG, DeprecationWarning)
```

Comment:
@chekunkov Looks like tests are disabled for pretty long period of time:
https://github.com/scrapinghub/python-hubstorage/commit/21f520b7bbc411eedbca9284bde0d57d72b46ffe
@dangra I guess we can restore it now?